docs/ko/docs/tutorial/path-operation-configuration.md

```diff
@@ -0,0 +1,98 @@
+# 경로 동작 설정
+
+*경로 동작 데코레이터*를 설정하기 위해서 전달할수 있는 몇가지 매개변수가 있습니다. 
+
+!!! warning "경고"
+    아래 매개변수들은 *경로 동작 함수*가 아닌 *경로 동작 데코레이터*에 직접 전달된다는 사실을 기억하십시오.
+
+## 응답 상태 코드
+
+*경로 동작*의 응답에 사용될 (HTTP) `status_code`를 정의할수 있습니다.
+
+`404`와 같은 `int`형 코드를 직접 전달할수 있습니다. 
+
+하지만 각 코드의 의미를 모른다면, `status`에 있는 단축 상수들을 사용할수 있습니다.
+
+```Python hl_lines="3  17"
+{!../../../docs_src/path_operation_configuration/tutorial001.py!}
+```
+
+각 상태 코드들은 응답에 사용되며, OpenAPI 스키마에 추가됩니다.
+
+!!! note "기술적 디테일"
+    다음과 같이 임포트하셔도 좋습니다. `from starlette import status`.
+
+    **FastAPI**는 개발자인 당신의 편의를 위해서 `starlette.status`와 동일한 'fastapi.status`를 제공합니다. 하지만 사실은 Starlette에서 직접 온것입니다.
+
+## Tags
+
+*경로 동작*에 태그를 달수도 있습니다, `tags` 매개변수를 `str`으로 이루어진 `list`의 형태로 전달하시면 됩니다 (보통은 한개의 `str`으로 이루어져 있습니다.)
+
+```Python hl_lines="17  22  27"
+{!../../../docs_src/path_operation_configuration/tutorial002.py!}
+```
+
+전달된 태그들은 OpenAPI의 스키마에 추가되며, 자동 문서 인터페이스에서 사용됩니다.
+
+<img src="/img/tutorial/path-operation-configuration/image01.png">
+
+## 요약과 기술
+
+`summary`와 `description`을 추가할수 있습니다.
+
+```Python hl_lines="20-21"
+{!../../../docs_src/path_operation_configuration/tutorial003.py!}
+```
+
+## 독스트링으로 만든 기술
+
+기술이 보통 길고 여러줄에 걸쳐있기 때문에, *경로 동작* 기술을 함수 <abbr title="함수안에 있는 첫번째 표현식으로, 문서로 사용될 여러 줄에 걸친 (변수에 할당되지 않은) 문자열"> 독스트링</abbr> 에 선언할수 있습니다, 이를 **FastAPI**가 독스트링으로부터 읽습니다.
```

Comment:
```suggestion
설명은 보통 길어지고 여러 줄에 걸쳐있기 때문에, *경로 동작* 기술을 함수 <abbr title="함수안에 있는 첫번째 표현식으로, 문서로 사용될 여러 줄에 걸친 (변수에 할당되지 않은) 문자열"> 독스트링</abbr> 에 선언할 수 있습니다, 이를 **FastAPI**가 독스트링으로부터 읽습니다.
```
I think `설명` is better than `기술` because the latter one confuses "description" with "technique"